frontend.py

```diff
@@ -27,18 +29,21 @@
 sites = ' '.join(list(patterns['username_patterns'].keys()))
 logos = json.dumps(patterns['logos'])
 signup = ' '.join(list(patterns['signup'].keys()))
-
+real_name = 'no'
 
```

Comment:
Here should be one more blank line like the [travis](https://travis-ci.org/manu-chroma/username-availability-checker/jobs/454067935#L1014) said

Reply:
k